src/HybridSolver.cpp

```diff
@@ -786,49 +378,82 @@ void HybridSolver::RecoverOriginalSolution(const mfem::Vector& HybridSol,
             HybridSol.GetSubVector(local_multiplier, mu_loc);
 
             // Compute u = (DMinvDT)^-1(f-DMinvC^T mu)
-            AinvDMinvCT_[iAgg].AddMult_a(-1., mu_loc, u_loc);
+            //AinvDMinvCT_[iAgg].AddMult_a(-1., mu_loc, u_loc);
+            tmp.SetSize(u_loc.size());
+            AinvDMinvCT_[iAgg].Mult(mu_loc, tmp);
+            u_loc -= tmp;
```

Comment:
I have no qualms with `AddMult`, but it was not yet implemented so I have just done it by hand for now.  It may require some work since all the different operators may need to be rewritten to apply `AddMult` instead of `Mult` and then redefine `Mult` in terms of `AddMult`.